index.html

```diff
@@ -200,7 +200,18 @@
       <li>a <dfn>performance entry buffer</dfn> to store
       <a>PerformanceEntry</a> objects. It is initially empty.
       </li>
+      <li>a list of <a>registered performance timeline emitter</a> objects. It is initially empty.</li>
     </ul>
+
+    <section>
+      <h2>Register <var>emitter</var> with <a>Performance Timeline</a></h2>
+      <p>An <var>emitter</var> represents a mechanism that can generate <a>PerformanceEntry</a> objects that are exposed to developers via the <a>Performance Timeline</a>. A <dfn>registered performance timeline emitter</dfn> consists of a:</p>
+      <ul>
+        <li><var>name</var> that MUST be unique.</li>
+        <li><var>performance entry buffer</var> that stores <a>PerformanceEntry</a> objects generated by the emitter, which is initially empty and has a <var>max-size</var> that, if unspecified, is set to 150.</li>
```

Comment:
At F2F we'd discussed buffering up to max buffer size OR load event? Not seeing any reference to load event here?

Reply:
@tdresser as FYI - looks like Max's CL might be blocked on this being clarified in the spec.

Reply:
@spanicker yep, good call, forgot to list this.. that's the behavior I was going to suggest.

- Each buffer has a max-size
- By default _all_ entries will be logged to their respective buffers up to onload event and size < max-size
- Some specs may also optionally choose to buffer after onload, at which point same size < max-size condition is enforced
  - if size =< max-size, new entries are dropped — i.e. we keep first N.

Reply:
> Does the buffering logic impact when events are added to the performance timeline in any way? 

No, when an entry is queued is up to the upstream spec. 

> When we say buffer up to the onload event, what are we actually comparing?

I was thinking with respect to when the entry is queued, but entry.startTime could work too.

Reply:
I think when the entry is queued is correct, especially in the face of user timing entries with arbitrary start times.

SGTM.